api/app/utils/task_metrics.py

```
@@ -408,3 +430,107 @@ def record_feedback_processing_failure(duration: Optional[float] = None) -> None
+    try:
+        from app.utils.task_metrics_persistence import get_persistence
+
+        persistence = get_persistence()
+        persistence.save_metrics(
+            {
+                "faq_extraction_last_run_status": FAQ_EXTRACTION_LAST_RUN_STATUS._value.get(),
+                "faq_extraction_messages_processed": FAQ_EXTRACTION_MESSAGES_PROCESSED._value.get(),
+                "faq_extraction_faqs_generated": FAQ_EXTRACTION_FAQS_GENERATED._value.get(),
+            }
+        )
+    except Exception:  # noqa: BLE001
```

Ruff (0.14.4)

Unused `noqa` directive (unused: `BLE001`)

Remove unused `noqa` directive

(RUF100)